### DISCUSSION TITLE
Moving kdl4j and kdl-gen into the organization

### DISCUSSION THREAD

**Discussion by hkolbeck** [kdl-org/kdl#333] [General]
I'm still happy to continue maintaining both, although they are admittedly on my back burners, but I think they'd be more visible and useful inside the org. 

kdl-gen (https://github.com/hkolbeck/kdl-gen) is a parser fuzzing utility I made that never got much attention, but I think it's a valuable tool to update quickly as any changes to the spec land, and to have multiple eyes on the code to make sure it really is producing valid KDL and is capable of producing the full set of valid KDL documents.

For anyone looking to lend a hand with kdl4j, the fuzzer has zeroed in on a few bugs. I solved a bunch, but a few are still of unclear origin. Invoking kdl-gen a few thousand times will definitely bring them up: https://github.com/hkolbeck/kdl4j/issues/8

💜

**Comment by zkat** [comment]
I'm happy to have this happen :)

**Reply by hkolbeck** [reply]
Awesome! I'll work on it in the next few days 💜

**Comment by hkolbeck** [comment]
They have been moved into the org:
- https://github.com/kdl-org/kdl-gen
- https://github.com/kdl-org/kdl4j

It's unclear if further tweaking of settings there is needed, I've never moved a repo before.

If this is going to be useful, the most pressing issue is probably getting additional eyes on `kdl-gen` before it's used by anyone to fuzz test a parser. An issue tracking that is here: https://github.com/kdl-org/kdl-gen/issues/3
